core/domain/event_services.py

```diff
@@ -73,11 +74,21 @@ class StatsEventsHandler(BaseEventHandler):
 
     EVENT_TYPE = feconf.EVENT_TYPE_ALL_STATS
 
+    @classmethod
+    def is_latest_version(cls, exp_id, exp_version):
```

Comment:
prefix with underscore to make it private

Reply:
Done.